code/src/components/Games.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react' 
+import { useSelector } from 'react-redux'
+import styled from 'styled-components/macro'
+
+import Header from './Header'
+import StartScreen from './StartScreen'
+import GameScreen from './GameScreen'
+import Loader from './Loader'
+
+const Games = () => {
+  const userName = useSelector(store => store.games.username)
+  const error = useSelector(store => store.games.error)
+  const loading = useSelector(store => store.games.loading)
+
+  return (
+    <MainWrapper>
+      <GameWrapper>
+      {error && `Ups, something went wrong, reason : ${error}`}
```

Comment:
Way to go on implementing an error message! 💯   Did you get it to work? Not sure how we can test it :)